src/sentry/sentry_metrics/indexer/limiters/cardinality.py

```diff
@@ -25,33 +29,29 @@
 @dataclasses.dataclass(frozen=True)
 class CardinalityLimiterState:
     _cardinality_limiter: CardinalityLimiter
-    _use_case_id: UseCaseKey
+    _metric_path_key: UseCaseKey
     _grants: Optional[Sequence[GrantedQuota]]
     _timestamp: Optional[Timestamp]
     keys_to_remove: Sequence[PartitionIdxOffset]
 
 
-def _build_quota_key(namespace: str, org_id: Optional[OrgId]) -> str:
+def _build_quota_key(use_case_id: UseCaseID, org_id: Optional[OrgId]) -> str:
     if org_id is not None:
-        return f"metrics-indexer-cardinality-{namespace}-org-{org_id}"
+        return f"metrics-indexer-cardinality-{use_case_id.value}-org-{org_id}"
     else:
-        return f"metrics-indexer-cardinality-{namespace}-global"
+        return f"metrics-indexer-cardinality-{use_case_id.value}-global"
 
 
 @metrics.wraps("sentry_metrics.indexer.construct_quotas")
-def _construct_quotas(use_case_id: UseCaseKey) -> Optional[Quota]:
+def _construct_quotas(use_case_id: UseCaseID) -> Optional[Quota]:
     """
     Construct write limit's quotas based on current sentry options.
 
     This value can potentially cached globally as long as it is invalidated
     when sentry.options are.
     """
-    if use_case_id == UseCaseKey.PERFORMANCE:
-        quota_args = options.get("sentry-metrics.cardinality-limiter.limits.performance.per-org")
-    elif use_case_id == UseCaseKey.RELEASE_HEALTH:
-        quota_args = options.get("sentry-metrics.cardinality-limiter.limits.releasehealth.per-org")
-    else:
-        raise ValueError(use_case_id)
+
+    quota_args = options.get(USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS[use_case_id])
```

Comment:
Probably should supply an additional default value here so we don't require all use cases to have it's own option name

Reply:
Also nit: should we be calling this
`USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS`
or
`USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTION_NAME`

Because I'm calling mine:
`USE_CASE_ID_WRITES_LIMIT_QUOTA_OPTION_NAME`

Reply:
I was thinking `USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS` but if you have a preference I'm fine either way.

And yep going to add the defaults now!

Reply:
added the default